ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -23,32 +22,27 @@ function BeamlineSetupContainer(props) {
     beamline,
     sampleChanger,
     sampleview,
-    sampleViewActions,
     uiproperties,
     setAttribute,
     stopBeamlineAction,
     sendCommand,
   } = props;
 
-  function handleSetAttribute(name, value) {
-    setAttribute(name, value);
-  }
-
   function renderBeamstopAlignmentOverlay() {
     const { hardwareObjects } = beamline;
     const motorInputList = [];
     let popover = null;
 
-    const motor = hardwareObjects.beamstop_alignemnt_x;
```

Comment:
There was a typo, but the hardware object is no longer called `beamstop_alignment_x` anyway but `diffractometer.beamstop_distance`.

This and the changes below fix the beamstop distance overlay as shown in the third screenshot (the cog icon + overlay with `OneAxisTranslationControl`).